src/theory/quantifiers/sygus/sygus_unif_rl.h

```diff
@@ -184,15 +202,19 @@ class SygusUnifRl : public SygusUnif
 
    private:
     /**
-     * reference to infered strategy for the function-to-synthesize this DT is
+     * reference to inferred strategy for the function-to-synthesize this DT is
      * associated with
      */
     SygusUnifStrategy* d_strategy;
+    /** index of strategy information of strategy node this DT is based on */
```

Comment:
Say more specifically what this index is referring to (index in StrategyNode::d_strats).